packages/zoe/tools/manualTimer.js

```diff
@@ -1,37 +1,79 @@
+// @ts-check
+
 import { E } from '@agoric/eventual-send';
+import { makeStore } from '@agoric/store';
+
+import './types';
+
+/* eslint-disable jsdoc/valid-types */
+/**
+ * @typedef {Object} ManualTimerAdmin
+ * @property {(msg?: string) => Promise<void>} tick
+ */
+/* eslint-enable jsdoc/valid-types */
 
-// A fake clock that also logs progress in tests.
+/**
+ * @typedef {ManualTimerAdmin & TimerService} ManualTimer
+ */
+
+/**
+ * A fake clock that also logs progress.
+ *
+ * @param {(...args: any[]) => void} log
+ * @param {Timestamp} [startValue=0]
+ * @returns {ManualTimer}
+ */
```

Comment:
Can we move these out of this file?

Reply:
I can put them in a new `internal-types.js` so that they don't pollute the namespace.